src/pages/WorldsDashboard/WorldsDashboard.tsx

```diff
@@ -26,11 +29,17 @@ type WorldsDashboardProps = { userId: UserId };
 export const WorldsDashboard: React.FC<WorldsDashboardProps> = ({ userId }) => {
   const { ownedVenues } = useOwnedVenues({ userId });
   const worlds = useWorlds();
+  const { ownWorlds } = useOwnWorlds();
+
+  const { isAdminUser: isMasterAdmin } = useAdminRole({ userId });
 
   const ownedUniqueWorldIds = useMemo(
-    // @debt should use uniq function of Lodash, or create our own in ./src/utils
-    () => [...new Set(ownedVenues.map(({ worldId }) => worldId))],
-    [ownedVenues]
+    () =>
+      uniq([
+        ...ownedVenues.map(({ worldId }) => worldId),
+        ...ownWorlds.map((world) => world.id),
+      ]),
+    [ownedVenues, ownWorlds]
   );
 
   // Firebase query where([world.id, 'in', ownedUniqueWorldIds]) has a limit of 10 items in ownedUniqueWorldIds. Because of that, it is filtered here
```

Comment:
Is this still needed for something?

Reply:
Yes, it's still relevant information

Reply:
Well, in that case, I think we can use few more tags in comments, like `@debt` but maybe `@workaround`, `@note` etc. Might confuse JSDoc software? :D